src/Simplex/Chat/protocol.md

```diff
@@ -2,26 +2,25 @@
 
 ## Design constraints
 
-- the transport message has a fixed size (8 or 16kb)
-- the chat message can have multiple parts/attachments
-- the chat message including attachments can be of any size
-- if the message is partially received, it should be possible to parse and display the received parts
+- the transport message has a fixed size (8 or 16kb), but the SMP agent will be updated to support sending messages up to maximum configured size (TBC - 64-256kb) in 8-16Kb blocks.
+- the chat message can have multiple content parts, but it should fit the agent message of the variable size.
+- one of the chat message types should support transmitting large binaries in chunks that could potentially be interleaved with other messages. For example, image preview would fit the message, but the full size image will be transmitted in chunks later - same for large files.
+- using object storage can be effective for large groups, but we will postpone it until content channels are implemented.
 
 ## Questions
 
 - should content types be:
   - limited to MIME-types
   - separate content types vocabulary
-  - both MIME types and extensions
+  - both MIME types and extensions (currently we support MIME (m.) and Simplex (x.) namespaces)
   - allow additional content types namespaces
 
 ## Message syntax
 
 The syntax of the message inside agent MSG:
 
 ```abnf
-agentMessageBody = message / msgContinuation
-message = [chatMsgId] SP msgEvent SP [parameters] SP [contentParts [SP msgBodyParts]]
+agentMessageBody = [chatMsgId] SP msgEvent SP [parameters] SP [contentParts [SP msgBodyParts]]
```

Comment:
`msgEvent SP [parameters]` is kind of free-form since according to protocol implementation optionality of parameters depends on specific msgEvent[Type], but it's ok for now I guess

Reply:
this is the general syntax for messages that parse to RawChatMessage, your comment is about semantic validity that is not defined in this syntax (and does not need to be).